ci/build_everest.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "##############   build everest dependency manager   #############"
+
+cd /checkout/everest-workspace/
+git clone https://github.com/EVerest/everest-dev-environment.git
+cd /checkout/everest-workspace/everest-dev-environment/dependency_manager
+python3 -m pip install .
+edm --config ../everest-complete-readonly.yaml --workspace /checkout/everest-workspace
```

Comment:
This just pull in the current main of everest, I don't think that's needed at all. Just cloning everest-core at the current PR branch should be enough (that pulls in everything needed)

Reply:
If i remove this I get the following error:
`Could not find EVerest dependency manager.  Please make it available in your PATH.`
And then the build stops with "error 2". 
Am I doing something wrong? Please specify more clearly how you would build everest-core without this.

Reply:
AH, github provided a bit too much context here. I would just
git clone https://github.com/EVerest/everest-core.git
here (on the current branch), or maybe there's a direct way of accessing the PRs everest-core here

Reply:
There would be a way to directly access the repository, but since the build folder will be put into the same base directory I can imagine all kinds of problems with that approach (considering this mapping would have to be on two layers: the runner is starting a docker container in which `everest-core` is being built, so the docker image would map onto the runner which in turn would map directly into the repository... Somehow this feels wrong.

Anyway, I still do not understand: However I have tried, I could only ever build everest-core, if the dependency manager and the utils were already checked out. Otherwise the build would always complain about missing tools...

Reply:
@hikinggrass : Today the edm-way stopped working, but building everest-core does not work (see runner log in the "Actions" tab).
I do not know if it is related, but first during the cmake run it complains that ` -- Could NOT find everest-cmake (missing: everest-cmake_DIR)`. Later make cannot find its target `install`. Please urgently advise.

Reply:
Yes, I really just meant replacing the edm --config line with a git clone of everest-core. Leaving the installing of ev-cli and edm untouched.
Interesting, looks like it's missing everest-cmake, which should be fetched automatically though.

Reply:
Resolved as discussed in meeting